CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-CU_SET_PATH("CMAKE_PREMIUM_DIR" "${CMAKE_CURRENT_LIST_DIR}")
+CU_SET_PATH("CMAKE_Npc_Services_DIR" "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
:rofl: @poszer

Reply:
>  @poszer

Like i said masterking xD

Reply:
@masterking32 always use the script in modules/ to create a default module !!